packages/aws-cdk/package.json

```diff
@@ -50,7 +50,8 @@
       "entryPoints": [
         "lib/index.js"
       ],
-      "sourcemap": "linked"
+      "sourcemap": "linked",
+      "minifyWhitespace": true
```

Comment:
Do we need to apply the same options to `cli-lib` ?

Reply:
Whitespace yes. `cli-lib` doesn't include a source map (on purpose).